### ISSUE TITLE
user-guide: merge/remove duplicate pages from new Installation subsection

### ISSUE THREAD

**Issue by dashohoxha** [iterative/dvc.org#701]
- [x] Move installation page to the top level of the docs (currently it is under the user-guide).
- [x] Remove https://dvc.org/doc/user-guide/plugins (which is now located under installation). See https://github.com/iterative/dvc.org/pull/695#pullrequestreview-302052801
- [x] Update https://dvc.org/doc/user-guide/development. See details in #700.
- [x] Simplify https://dvc.org/doc/get-started/install and maybe merge it with the next section (https://dvc.org/doc/get-started/initialize). See https://github.com/iterative/dvc.org/pull/695#pullrequestreview-302050578
- [x] Find all links to the removed pages and redirect them to the new pages.

Depends on: #695
Related: #700 , #660

**Comment by jorgeorpinel** [comment]
Can we include adding relevant updating notes to this PR? See https://github.com/iterative/dvc.org/pull/695#pullrequestreview-302330591

**Comment by shcheklein** [comment]
@jorgeorpinel I would keep it as a separate ticket. It's not a simple one :)

**Comment by jorgeorpinel** [comment]
OK, moved to #702

**Comment by jorgeorpinel** [comment]
>  Remove https://dvc.org/doc/user-guide/development (which is now located under installation). Same as #700?

Duplicates #700 but OK, we can consolidate the issues here 🙂

**Comment by jorgeorpinel** [comment]
> Remove https://dvc.org/doc/user-guide/development

I'm addressing this in https://github.com/iterative/dvc.org/pull/695/commits/1774355eca250c4fb2779634e167b865ca4bbee3.

**Comment by jorgeorpinel** [comment]
> Update https://dvc.org/doc/user-guide/development 

This part is not done right. See https://github.com/iterative/dvc.org/issues/701#issuecomment-542929946. It should be re-done.

**Comment by jorgeorpinel** [comment]
(Re-done by re-applying the changes in https://github.com/iterative/dvc.org/commit/1774355eca250c4fb2779634e167b865ca4bbee3 which were reverted as explained in https://github.com/iterative/dvc.org/pull/695#pullrequestreview-303630301.)

**Comment by shcheklein** [comment]
@jorgeorpinel since the PR has been merged you need to create a new with these changes probably?

**Comment by dashohoxha** [comment]
> It should be re-done by re-applying the changes

@jorgeorpinel The necessary changes have been applied correctly, please check it again. So, let's close this issue (an open another one if needed).

**Comment by jorgeorpinel** [comment]
Yes @shcheklein, a new PR is needed to fix this. I am happy to do this work myself **again**, but do we agree that undoing other people's work is confusing and unproductive? I could've missed this problem for example, and it would be totally lost.

In this particular case, I was asked to review that PR in https://github.com/iterative/dvc.org/pull/695#event-2715414398, then I was asked for help on it in https://github.com/iterative/dvc.org/pull/695#issuecomment-542884588, and then when I contributed to the branch, my work was reverted, which we noticed, but our comments were ignored.

Can we clarify what **good teamwork** should look like for further interactions? In fact this is happening again in https://github.com/iterative/dvc.org/pull/732#issuecomment-546037236 right now.
